desktop/apps/loyalty/index.js

```diff
@@ -1,4 +1,4 @@
 const express = require('express')
 
 const app = module.exports = express()
-app.use('/loyalty', require('@artsy/reaction-force').default.apps.loyalty)
+app.use('/loyalty', require('@artsy/reaction-force/dist/apps/loyalty/server').default)
```

Comment:
It did, for the same reason we have to require components from the `dist` folder on the client-side (but for the opposite reason). All the server stuff (`artsy-passport`, etc) breaks the client-side requires, and now `history` will break the server-side requires (if we don't require from the direct path).

I still maintain that reaction-force is not the place to build entire apps but rather the components that will be used to build an app. The gluing should be done outside reaction-force IMO.